daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -75,7 +76,9 @@ runDamlStudio overwriteExtension remainingArguments = do
     let vscodeExtensionTargetDir = vscodeExtensionsDir </> vscodeExtensionName
     when overwriteExtension $ removePathForcibly vscodeExtensionTargetDir
     installExtension vscodeExtensionSrcDir vscodeExtensionTargetDir
-    exitCode <- withCreateProcess (proc "code" ("-w" : remainingArguments)) $ \_ _ _ -> waitForProcess
+    -- Note that it is important that we use `shell` rather than `proc` here as
+    -- `proc` will look for `code.exe` in PATH which does not exist.
```

Comment:
The reason is that `code` is really `code.cmd` which can only be run through a shell.